src/Dashboard/Data/FunctionIndexReader.cs

```diff
@@ -70,7 +70,8 @@ public DateTimeOffset GetCurrentVersion()
             return _versionMapper.GetVersion(blob.Metadata);
         }
 
-        public IResultSegment<FunctionIndexEntry> Read(int maximumResults, string continuationToken)
+        // Classic logging ignores hostName filter. 
```

Comment:
Is this for the top level Dashboard view showing all functions across all hosts? I.e., the same function name might be shown twice in the table, although the actual invocation logs ARE separated by host id.